actnow/petitions/models.py

```diff
@@ -69,3 +72,10 @@ def __str__(self):
 
     class Meta:
         unique_together = ("petition", "signatory")
+
+
+class Issue(TimestampedModelMixin):
+    link = models.URLField(_("link"))
```

Comment:
1. Shouldn't this be unique?
2. Can we also clean them up e.g. make sure all urls end with /, all have http or https, etc. With end results being we reduce the likelyhood of having the same url entered multiple times.

Reply:
@kilemensi  for 
>Can we also clean them up e.g. make sure all urls end with /, all have

Is this in addition to the default [validator](https://github.com/django/django/blob/main/django/db/models/fields/__init__.py#L2307)? 
I case a URL doesn't end with a `/` we should add it so the only cleaning we will be doing is something like
```
def check_path(path):
    if not path.endswith("/"):
        path += "/"
    return path

```

Reply:
1. If the default validator already does what we want (the above), then we don't need to do anything.
2. We need to be careful with `/`, if url is `something.domain/path`, then yes. But if it's `something.domain/path?query` then `/` should be added at the end of `path` and not `query` i.e. `something.domain/path/?query` or just leave the url as is.

Reply:
@kilemensi  how about just checking if it ends with a `/` and removing it instead of adding it? URLs will work without the trailing `/` right?

Reply:
Mostly yes but not always (e.g. before v9.5, [Nextjs](https://nextjs.org/docs/api-reference/next.config.js/trailing-slash) didn't handle this properly).

But yes, which ever is simpler for us to implement.